docs/3.0/common/reloadable.md

```diff
@@ -0,0 +1,31 @@
+:::option Reloadable
+
+<VersionReq version="3.3.6" />
+
+The reloadable option adds a reload icon next to the association title so users can easily reload just that turbo-frame instead of doing a full page reload.
+
+#### Usage
+To enable the `reloadable` feature, provide a block to the reloadable option when defining the field. Within this block, specify conditions under which the reloadable should be displayed.
+The reloadable option supports the following structure:
+- **Default value**: `false`
+- **Possible values**:
+  - `true`: Enables the reloadable feature unconditionally.
+  - `false`: Disables the reloadable feature.
+  - `Proc`: Accepts a block that evaluates conditions for displaying the reloadable feature.
+
+```ruby
+field :reviews, as: :has_many,
+  reloadable: -> {
+    current_user.is_admin?
+  }
+```
+In the above example, the reloadable will be visible if the current_user is an admin.
+
+```ruby
```

Comment:
Let's display the boolean version first and then the block version.

Reply:
Sure